main/src/CoffeeMachineSimulator.java

```diff
@@ -0,0 +1,199 @@
+import java.util.Scanner;
+
+public class CoffeeMachineSimulator {
+
+    int water = 400;
+    int milk = 540;
+    int coffeeBeans = 120;
+    int disCups = 9;
+    int money = 550;
+    String buy = "buy";
+    String fill = "fill";
+    String take = "take";
+    String exit = "exit";
+    String remaining = "remaining";
+    String back = "back";
+    int mode = 1;
+
+
+    // method checkMethod check coffee machine mode, and call appropriate method
```

Comment:
@Edvard777 please try to update comments as I show yesterday

Reply:
done